packages/backend/src/generated/routes.ts

```diff
@@ -12330,10 +12330,11 @@ export function RegisterRoutes(app: Router) {
         async function ValidationController_post(
             request: ExRequest,
             response: ExResponse,
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
             next: any,
         ) {
             // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
-
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
@rephus this is a generated file and we need to ignore this from the eslint but I am not sure what we can do with typescript compiler. I had a different suggestion for this kind of rule adoptions -> https://github.com/lightdash/lightdash/issues/3790#issuecomment-2604998824 not sure if you had the chance to have a look

Reply:
oh yeah, we don't need to add // eslint to routes or swagger.